staging/src/k8s.io/apiserver/pkg/server/filters/timeout.go

```diff
@@ -91,14 +91,19 @@ func (t *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	done := make(chan struct{})
+	result := make(chan interface{})
 	tw := newTimeoutWriter(w)
 	go func() {
+		defer func() {
+			result <- recover()
```

Comment:
so easy to start a gofunc, so easy to forget to recover panics in it.

Reply:
Now go through all go funcs in Kube and prepare for new grey hair.